splunk_connect_for_snmp/inventory/loader.py

```diff
@@ -116,9 +119,9 @@ def load():
                     task_config = gen_walk_task(ir)
                     periodic_obj.manage_task(**task_config)
 
-            except:
+            except Exception as e:
                 inventory_errors = True
-                logger.exception(f"Exception processing {source_record}")
+                logger.error(f"Exception raised for {address}: {e}")
```

Comment:
exception() will give us stacktrace while error() will not

Reply:
brought back